vayesta/dmet/__init__.py

```diff
@@ -21,9 +21,9 @@
 
 def DMET(mf, *args, **kwargs):
     """Determine restricted or unrestricted by inspection of mean-field object"""
-    if isinstance(mf, pyscf.scf.uhf.UHF):
+    if isinstance(mf, pyscf.scf.uhf.UHF) or isinstance(mf, pyscf.pbc.scf.uhf.UHF) or isinstance(mf, pyscf.pbc.scf.kuhf.KUHF):
```

Comment:
FYI, `isinstance` accepts a tuple of possible classes as a second argument

Reply:
Oh, I didn't know that. Thanks!